### DISCUSSION TITLE
model has no textures.

### DISCUSSION THREAD

**Discussion by laurentiu04** [google/model-viewer#4203] [Q&A]
So I downloaded the .blend file, then I exported the model as a .glb file from blender. I succesfully inserted it into the site, but it's all white. The model is stored in src/assets/3D, where the folder with the textures is stored as well.

I'm using react btw.

**Comment by WadeMorrison** [comment]
Does the model show up correctly when you put it in the [editor](https://modelviewer.dev/editor/)? 

If not, you may need to make sure the "Materials" is set to "Export" in the glb/gltf settings, anything else will make it white/unshaded. If that is correct, then you may need to ensure your material nodes are set up correctly or that there even is a material applied to it.
